plugins/steelseries/fu-steelseries-fizz.c

```diff
@@ -192,13 +174,16 @@ fu_steelseries_device_command_and_check_error(FuDevice *device, guint8 *data, GE
 	return fu_steelseries_device_command_error_to_error(cmd, err, error);
 }
 
-static gchar *
-fu_steelseries_fizz_version(FuDevice *device, GError **error)
+gchar *
+fu_steelseries_fizz_version(FuDevice *device, gboolean tunnel, GError **error)
```

Comment:
fu_steelseries_fizz_ensure_version perhaps?

Reply:
I would prefer `fu_steelseries_fizz_get_version()` if following the reviews above.